clients/privacy-center/components/modals/consent-request-modal/ConsentRequestModal.tsx

```diff
@@ -19,7 +19,11 @@
   );
 
   const successHandler = useCallback(() => {
-    router.push("consent");
+    let consentRoute = "/consent";
+    if (router.query.customPropertyPath) {
+      consentRoute = `/${router.query.customPropertyPath}/consent`;
+    }
+    router.push(consentRoute);
```

Comment:
Hmmm I see why this may cause a warning. However, if that customPropertyPath were to have a malicious content, the page would render a 404 page. It will only render if we get a result back from the _get properties by path_ endpoint. Also, the resulting history push starts with a '/', so it should stay on the domain.